aspnetcore/migration/Logging-nonaspnetcore.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Migrate from Microsoft.Extensions.Logging 2.1 to 2.2 or 3.0
+author: pakrym
+description: This article outlines the common steps for migrating a non-ASP.NET Core application that uses Microsoft.Extensions.Logging 2.1 to Microsoft.Extensions.Logging 2.2 or 3.0.
```

Comment:
Again ... "Learn ..." "Discover ..." ... this came down from some APEX guidance ages ago. I think while you were away we went thru and made a huge sweep to update all of the topics to that format.